index.html

```diff
@@ -37,8 +37,128 @@ <h1>I'm Sohail Glad to see you!</h1>
+                <div class="card-work work">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio-1.png" alt="">
+                <div class="card-work work">
+                    <h3>Tonic</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+            <div class="works">
+                <img src="./images/Snapshoot Portfolio-2.png" alt="">
+                <div class="card-work work">
+                    <h3>Multi-Post <br> Stories</h3>
+                    <ul class="proj">
+                        <li><a href="#">CANOPY</a></li>
+                        <li><div class="dot"></div></li>
+                        <li>Back End Dev</li>
+                        <li><div class="dot"></div></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="pro">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="language">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button><a href="#">See Project</a></button>
+                </div>
+            </div>
+        </div>
+<!-- ------------------------------------------------about me--------------------------------         -->
+        <div class="details">
+            <h1>About <br> Myself</h1>
```

Comment:
- [ ] It would be better, because we already have an h1 tag, to use h2 for the About Myself title, in order to respect the [HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) :recycle: